code/modules/mob/living/carbon/human/human.dm

```diff
@@ -679,6 +679,28 @@
 		else
 			return ""
 
+/mob/living/carbon/human/Hear(message, atom/movable/speaker, var/datum/language/speaking, raw_message, radio_freq)

```

Comment:
Do the proc override, copypasting an entire proc to make a small change isn't necessary